bool robot_det
bool robot_act
bool Estop_det
bool Estop_act
bool door_det
bool door_act
bool curtain_det
bool curtain_act
bool UI_det
bool UI_act
bool vision_det
bool vision_act

